rust/kcl-lib/src/execution/mod.rs

```diff
@@ -2167,6 +2167,80 @@ notPipeSub = 1 |> identity(!%))";
         // let notInfinity = !Infinity
     }
 
+    #[tokio::test(flavor = "multi_thread")]
+    async fn test_start_sketch_on_invalid_kwargs() {
+        let ast = r#"
+sketch001 = startSketchOn(XY)
+  |> startProfile(at = [-5, -5])
+  |> xLine(length = 10, tag = $a)
+  |> yLine(length = 10, tag = $b)
+  |> xLine(length = -10, tag = $c)
+  |> close()
+
+cube001 = extrude(sketch001, length = 10)
+
+sketch002 = startSketchOn(cube001, normalToFace = a, face = a)
+"#;
+        assert_eq!(
+            parse_execute(ast).await.unwrap_err().message(),
+            "You cannot give both `face` and `normalToFace` params, you have to choose one or the other.".to_owned(),
+        );
+
+        let ast = r#"
+sketch001 = startSketchOn(XY)
```

Comment:
Could you actually please move these into a file under `kcl-lib/tests/inputs/` and then use `std::fs::read_to_string` to load them? That way we can easily format or fix up the KCL. It's much easier to use this with CLI tools than via searching in Rust code.